### ISSUE TITLE
Image feature extraction and comparison

### ISSUE THREAD

**Issue by demonlov** [bytedeco/javacv#1800]
How to use image eigenvalues and extraction, I need some sample code, I didn't find the detector of OpenCV

**Comment by saudet** [comment]
If you have some sample code in C++, we can start from there.

**Comment by demonlov** [comment]
> If you have some sample code in C++, we can start from there.

No, I don't have C + + code. I need javacv for image feature extraction and comparison

**Comment by demonlov** [comment]
For mat in javacv, I can't find the get () method. How to use double [] grayarr = mat.get(i, j) in opencv ;  This method

**Comment by saudet** [comment]
We can use the Java API of OpenCV with JavaCV as well:
http://bytedeco.org/javacpp-presets/opencv/apidocs/org/opencv/core/Mat.html#get-int-int-
